telegram/ext/_application.py

```diff
@@ -597,7 +604,10 @@ def run_polling(
         .. seealso::
             :meth:`initialize`, :meth:`start`, :meth:`stop`, :meth:`shutdown`
             :meth:`telegram.ext.Updater.start_polling`, :meth:`telegram.ext.Updater.stop`,
-            :meth:`run_webhook`
+            :meth:`run_webhook`,
+            `Your First Bot <https://github.com/\
+            python-telegram-bot/python-telegram-bot/wiki/Extensions-–-Your-first-Bot>`_,
+            :any:`Echobot Example <examples.echobot>`
```

Comment:
listing this in the class docstring is enough IMO

Reply:
Done